### ISSUE TITLE
Missing CDT Branding of the Eclipse CDT Package

### ISSUE THREAD

**Issue by azoitl** [eclipse-cdt/cdt#962]
**Is your feature request related to a problem? Please describe.**
When I download the CDT Package from the [Eclipse Package](https://www.eclipse.org/downloads/packages/) and unzip I don't see anything that I have CDT (see screenshot below). It does not get much better when I start it. The title bar still says Eclipse IDE and the icon is the Eclipse IDE icon. 

I think this is very bad from a user experience perspective and it should be corrected.

**Describe the solution you'd like**
I would expect that the executable, the icon, the title and maybe more, is branded with Eclipse CDT. 

As I have experience with that topic I'm happy to provide the required branding. I would just need a pointer with the sources of the current CDT logo, the desired branding names, and if the branding is set here or in the EPP package.

As proposal and starting point:
  - Window Title bar text: Eclipse CDT
  - Executable name: eclipse_cdt(.exe)

**Additional context**
File system of a freshly downloaded CDT package:

![Screenshot From 2024-12-12 13-26-17](https://github.com/user-attachments/assets/154434f0-5b62-4ad2-a012-b4abd1a3a3bc)

**Comment by jld01** [comment]
Hi @azoitl, thank you for your feedback. Do you see this as a source of confusion for new users? What do you think a new user will be expecting to see when they unzip the package?

A counter-argument to further customization of the EPP package is that the Eclipse IDE is simply a container for Eclipse features. Some users may add additional features following initial installation. The installation may become much more than an IDE for C/C++ development.

**Comment by azoitl** [comment]
> Hi @azoitl, thank you for your feedback. Do you see this as a source of confusion for new users? What do you think a new user will be expecting to see when they unzip the package?

Yes I see it as a source for confusion for new users. Not only after unziping but even more when starting. What I realized after writing my post is that the new users is then confronted with the default Eclipse IDE splash. Only when a workspace is chosen and the window is coming up the welcome screen gives some hints that this may be the right package the was started. 

When unzipping I would expect something that is indicating this could be CDT. But it is not only about unziping. It is also when creating shortcuts, or as I do it add it why *.desktop files to my Gnome start menu. When I look for eclipse and have one or more packages (in my case often 3 to 4 different, see below why) I get several times the same icon and I have to read in the description or hover what are the different packages. This is slower then just looking for an icon. I for me have fixed this by creating my own local CDT icon, but that does not scale and is effort that I would like to spend for other things.
 
> A counter-argument to further customization of the EPP package is that the Eclipse IDE is simply a container for Eclipse features. Some users may add additional features following initial installation. The installation may become much more than an IDE for C/C++ development.

I know that. As a long time Eclipse users (app. 20y) I did that for a long time. However I stopped that because I noticed that the individual packages are more lightweight less options, they also feel slightly faster (which may be wrong). Now I use individual packages for my Eclipse tasks (e.g., Java, CDT) and add there all extensions I need for these tasks. 

But this now opens the question how Eclipse IDE and CDT would like to handle that topic from a branding and brand development perspective. Is the brand Eclipse IDE and all the other packages are some kind of flavor or are the individual packages individual brands by themselves. We can see both options with our market companions:

The first one is VSCode, but with the big difference that you download VS Code and only VS Code and everything you would like to do with it are extensions. 

The other extreme is Jetbrains they have tailored individual products with a strong branding of every one. 

We are somehow in the middle and I'm not sure if this is good or bad. 

What I know that for me it feels strange to download Eclipse CDT but it does not feeling like getting CDT.

**Comment by jonahgraham** [comment]
> What I know that for me it feels strange to download Eclipse CDT but it does not feeling like getting CDT

To add to the confusion, you don't download Eclipse CDT, the products available today are _Eclipse IDE for C/C++ Developers_ or _Eclipse IDE for Embedded C/C++ Developers_ (cpp or embed-cpp are their code names).  These are both managed by the [EPP project](https://github.com/eclipse-packaging/packages), and decisions about branding should be made there, but probably in conjunction with Eclipse IDE WG and Eclipse Foundation as they have traditionally owned much of the marketing of the Eclipse IDE. I don't think that _Eclipse IDE for C/C++ Developers_ can/should go out on its own and rebrand, unless we also rebrand the other packages in EPP too.

**Comment by jonahgraham** [comment]
I feel my comment may have been a little negative - I encourage this effort, but it requires someone with enough time and passion to see it through, both technically and community building. I don't have that energy for this topic at this point in time, but I want to encourage you to pursue this if you do have that energy. Tracy did some of this work along with others many years ago to ditch eclipse release names (see the most popular post on our blog https://kichwacoders.com/2016/04/28/why-its-time-to-kill-the-eclipse-release-namesneon-oxygen-etc/).
